pacfetcher_test.go

```diff
@@ -22,6 +22,19 @@ func pacjsHandler(pacjs string) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) { w.Write([]byte(pacjs)) }
 }
 
+type pacServerWhichFailsOnFirstTry struct {
```

Comment:
If this type were `pacServerWhichFailsFirstNTries` and was initialised with a count, and decremented that count erroring until the count was 0, you could also use it to test the untest case of 2+ failures. But that wasn't tested before so no real need to test it now. Up to you.

Reply:
I think the interesting equivalence classes are: 1) consistently failing downloads, 2) consistently succeeding downloads, and 3) failing downloads that at some point start to work. Having 1, 2 or n initial failures before the first successful download all seem to be the same as the third class of tests to me.

Also, I'm a little hesitant to do this because the timeout is 2 seconds, not a fixed number of failures, and if we go to a high-enough number of fixed failures then we'll end up with a flaky test.